src/include/wiredtiger.in

```diff
@@ -270,7 +270,8 @@ struct __wt_cursor {
 	 */
 	/*!
 	 * Return the ordering relationship between two cursors: both cursors
-	 * must have the same data source and have valid keys.
+	 * must have the same data source and have valid keys. (When testing
+	 * only for equality, WT_CURSOR::compare_equality may be faster.)
```

Comment:
That should be WT_CURSOR::compare_equal.  Is there other documentation describing return values or if/when there is any reason a user may want to use compare instead for anything other than less than or greater than now?  Does this just return 0 or non-zero?